### ISSUE TITLE
Share.refresh() method in 1.1.3 raises exception

### ISSUE THREAD

**Issue by lebedov** [lukaszbanasiak/yahoo-finance#11]
When I tried to refresh the data associated with the 'GOOG' share, I observed the following:

```
>>> import yahoo_finance
>>> s=yahoo_finance.Share('GOOG')
>>> s.get_price()
'532.42'
>>> s.refresh()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/lev/miniconda/envs/DEFAULT/lib/python2.7/site-packages/yahoo_finance/__init__.py", line 141, in refresh
    self.data_set = self._fetch()
  File "/home/lev/miniconda/envs/DEFAULT/lib/python2.7/site-packages/yahoo_finance/__init__.py", line 178, in _fetch
    data = super(Share, self)._fetch()
  File "/home/lev/miniconda/envs/DEFAULT/lib/python2.7/site-packages/yahoo_finance/__init__.py", line 133, in _fetch
    data = self._request(query)
  File "/home/lev/miniconda/envs/DEFAULT/lib/python2.7/site-packages/yahoo_finance/__init__.py", line 122, in _request
    raise YQLQueryError(response['error']['description'])
yahoo_finance.YQLQueryError: Query failed with error: ""Query syntax error(s) [line 1:29 expecting table got 'where',line 1:36 expecting equalityExpression got '=']"".
```

I'm using Python 2.7.9 on Ubuntu 14.04.1.

**Comment by lukaszbanasiak** [comment]
@lebedov I confirm error. Fix it soon.

**Comment by lukaszbanasiak** [comment]
Fixed in 97c4358 and released in 1.1.4 version
